### DISCUSSION TITLE
[Idea] Add latest download link for each download source

### DISCUSSION THREAD

**Discussion by Jason-Linux** [glpi-project/glpi-agent#514] [Ideas]
### Your idea

Hi,

To maintain glpi agent uptodate, i would like to script installation and maj with the latest release.
Can you add this in download link.

Best Regards

**Comment by g-bougard** [comment]
Hi @Jason-Linux 

if you're scripting, you'll find a tip to use github public api here in my own comment: #174
The purpose is to catch datas related to latest release with this api and make your script decides what to do.

**Reply by Jason-Linux** [reply]
thanks for your fast answer

How you choose for which os ?
I'm sorry im not friendly with this.

**Reply by g-bougard** [reply]
The package base name won't change so the only thing you're missing is what is the latest released version. This is the purpose of calling github API.
From that, you can:
 - guess package filename and download url
 - compare the version with a stored one or check if the current installed version is the latest
